source/agora/test/ValidatorRecurringEnrollment.d

```diff
@@ -234,44 +277,46 @@ unittest
     auto blocks = node_1.getBlocksFrom(0, 2);
     assert(blocks.length == 1);
 
-    auto sleep_node_1 = nodes[$ - 1];
-    auto sleep_node_2 = nodes[$ - 2];
+    auto sleep_node_1 = nodes[$ - 1]; // node #5
+    auto sleep_node_2 = nodes[$ - 2]; // node #4
 
     // Approach end of the cycle
-    network.generateBlocks(Height(GenesisValidatorCycle - 2));
+    network.generateBlocks(Height(GenesisValidatorCycle - 3));
 
-    // Make 2 nodes sleep
+    // Make 2 nodes sleep (they will not enroll as they are also DelayedEnrollNodes)
     sleep_node_1.ctrl.sleep(60.seconds, true);
     sleep_node_2.ctrl.sleep(60.seconds, true);
 
     network.generateBlocks(iota(GenesisValidators - 2),
         Height(GenesisValidatorCycle - 1));
 
-    // Wake one up right before cycle ends
+    // Wake up node #4 right before cycle ends
     sleep_node_2.ctrl.sleep(0.seconds);
     // Let it catch up
-    network.expectHeightAndPreImg(iota(0, GenesisValidators - 1),
+    network.expectHeightAndPreImg(only(GenesisValidators - 2), // node #4
         Height(GenesisValidatorCycle - 1), network.blocks[0].header);
 
-    network.generateBlocks(iota(GenesisValidators - 1),
+    network.generateBlocks(iota(GenesisValidators - 1), // nodes #0 -> #4
```

Comment:
```suggestion
    network.generateBlocks(iota(GenesisValidators - 1), // nodes #0 .. #4
```
At the first sight, I'm thinking what the meaning of `->` is. But I would suggest removing this kind of comment because if we make change the line, we could forget to modify the comment.

Reply:
Updated